spec/lib/miq_automation_engine/service_methods/miq_ae_service_manageiq-providers-vmware-infra_manager-provision_spec.rb

```diff
@@ -311,6 +311,21 @@ def invoke_ae
       end
     end
 
+    context "storage_profile" do
+      before(:each) do
+        @cs = FactoryGirl.create(:storage_profile, :name => "Test StorageProfile")
+        allow_any_instance_of(MiqProvisionVirtWorkflow).to receive(:allowed_storage_profiles).and_return(@cs.id => @cs.name)
+      end
+
+      it "#eligible_storage_profile" do
```

Comment:
Should be `#eligible_storage_profiles`.  Missing the "s".

You will also need to test the corresponding `set_storage_profile` method which is created by `expose_eligible_resources :storage_profiles`